sources/packages/backend/apps/workers/src/zeebe/zeebe-transport-strategy.ts

```diff
@@ -30,12 +42,38 @@ export class ZeebeTransportStrategy
         // be retrieved from the workflow as per Camunda best practices.
         fetchVariable: workerOptions?.fetchVariable ?? [],
         taskType,
-        taskHandler: async (job) => handler(job),
+        taskHandler: async (job) => this.workerHandlerWrapper(handler, job),
       });
     });
     callback();
   }
 
+  /**
+   * Wrapper to allow global handling of all the jobs executed.
+   * @param jobHandler job method to be executed.
+   * @param job job information.
+   * @returns job result.
+   */
+  private async workerHandlerWrapper(
```

Comment:
This is the wrapper which handles all the worker jobs right? I guess it is mainly to handle the cross-cutting concerns like logging and error handling.

Reply:
Yes, that is the idea. For anything that we would like to handle globally, for instance, we can capture unhandled exceptions, check if they are DB connectivity issues, and change the time to retry to something different.